src/Microsoft.Framework.Runtime.Hosting/project.json

```diff
@@ -4,6 +4,7 @@
     "dependencies": {
         "Newtonsoft.Json": "6.0.6",
         "NuGet.ProjectModel": "3.0.0-*",
+        "NuGet.RuntimeModel": "3.0.0-*",
```

Comment:
The object model for representing Runtimes, reading `runtime.json` and expanding a runtime to it's compatible runtimes. It's basically just Pocos and Json parsing

https://github.com/NuGet/NuGet3/tree/dev/src/NuGet.RuntimeModel